main.py

```diff
@@ -2,6 +2,40 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # Create genres
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
```

Comment:
DRY - don't repeat yourself.
Look at checklist for this task, and stick to best practice, what if we have 10 or 15 genres?